### DISCUSSION TITLE
Ability to hide Stories (TTV)

### DISCUSSION THREAD

**Discussion by smol-fi** [night/betterttv#6997] [Ideas]
Really not into Stories taking up significant screenspace, particularly on the Follwing page:
![image](https://github.com/user-attachments/assets/4070290e-39ee-45d7-aef6-3f798af0f103)

The sidebar has a toggle for showing Stories as well. Preferably a toggle to disable either/both, in the event someone does care about them and wants them accessible via one option. If someone noticed any annoyances related to them that I didn't, please add them below.
![image](https://github.com/user-attachments/assets/3733dc4a-4016-4dbf-937a-ff3ee8574cb2)

**Comment by odnoyabr** [comment]
yeah they've started trying to shove these into the desktop clients. for the _Following_ page, these ublock filters seem to work:

```
! Aug 15, 2024 https://www.twitch.tv
www.twitch.tv##.storiesLeftNavSection--csO9S.dcyYPL.Layout-sc-1xcs6mc-0
www.twitch.tv##.lcFJxY.Layout-sc-1xcs6mc-0
```

larger HTML context if useful:

[stories_main_segment.html.txt](https://github.com/user-attachments/files/16632142/stories_main_segment.html.txt)

[stories_sidebar.html.txt](https://github.com/user-attachments/files/16632143/stories_sidebar.html.txt)

**Reply by smol-fi** [reply]
i'll add with my own filters, they changed some of the classes recently:
```
www.twitch.tv##.dKitkM.Layout-sc-1xcs6mc-0
www.twitch.tv##.storiesLeftNavSection--csO9S
! last one looks a bit stupid if sidebar is collapsed
www.twitch.tv##.storiesLeftNavSectionCollapsedButton--txKvw
```

**Comment by ElectroOne** [comment]
On PC, (Any Browser) if you have Ublock Origin Installed, just Right Click the Stories on the Top Left of the screen, then select Block Element. Then POOF, it is Permanently Gone for you. This would likely work on any PC Browser Adblocker that has an Element Blocker built into it.

**Reply by ChaosElixir** [reply]
<img width="327" height="179" alt="image" src="https://github.com/user-attachments/assets/b56ca7ce-ddcc-4e0c-b03a-36c14b8191fc" />

<img width="319" height="163" alt="image" src="https://github.com/user-attachments/assets/9d3c7b60-5059-449a-808d-d1eee1389d47" />

If you block the stories that way using ublock it also blocks other parts of the site. For example, that filter is also causing these twitch rewards to not show any text at the top of a stream (first image with filter, second without). Really would just be nice to have an option to disable it in bttv (if that's possible).

**Comment by dclstn** [comment]
Closing this discussion, as this feature exists today.
